.github/CODEOWNERS

```diff
@@ -36,32 +26,20 @@
 
 /packages/kokkos-kernels/    @trilinos/kokkos-kernels
 
-/packages/komplex/           @trilinos/komplex
-
-#/packages/meshinggenie/      @trilinos/
-
-#/packages/minitensor/        @trilinos/
```

Comment:
minitensor is still package, but is there a difference between having @trilinos/ versus not having anything?